gtclang/README.md

```diff
@@ -1,75 +1,48 @@
 gtclang <br/> <a target="_blank" href="http://semver.org">![Version][Version.Badge]</a><!-- <a target="_blank" href="https://travis-ci.org/MeteoSwiss-APN/gtclang">![Travis status][TravisCI.Badge]</a>--> <a target="_blank" href="https://opensource.org/licenses/MIT">![License: MIT][MIT.License]</a>
 ===========
 
-**gtclang** is a source-to-source compiler with a high level DSL (Domain-Specifc Language) to execute stencil-like computations on a variety of different parallel architectures. The target applications for gtclang are regional weather models (regular grids) as well as global weather and climate simulations (irregular grids). gtclang is built on top of the [LLVM/Clang](https://clang.llvm.org/) compiler framework and [Dawn](https://github.com/MeteoSwiss-APN/dawn) and produces highly optimized C++ source code for the gridtools library.
+## Introduction
+
+**gtclang** is part of a compiler toolchain for developing high-level DSLs for geophysical fluid dynamics models. **gtclang** is the frontend part of the toolchain that processes a high level DSL and turns it into an intermediary representation (SIR), to be processed by dawn. See the top level `README` for a detailed description. Consider the following code snippet for an example stencil that performs two Laplacian operators in succesion using finite differences:
 
 [![gtclang](https://raw.githubusercontent.com/MeteoSwiss-APN/gtclang/master/docs/images/hd.png)](https://github.com/MeteoSwiss-APN/gtclang/releases)
 
 ## Building
 
-gtclang depends on [Dawn](https://github.com/MeteoSwiss-APN/dawn) as well as [Clang](https://clang.llvm.org/) (3.8.0), the generated code further requires gridtools and optionally [CUDA](https://developer.nvidia.com/cuda-downloads). The build process requires a C++11 compiler and [CMake](https://cmake.org/). To build all these dependencies with a single CMake invocation, it is highly recommended to use the meta repository [gtclang-all](https://github.com/MeteoSwiss-APN/gtclang-all).
-
-
-If you wish to directly compile the library, make sure CMake can find Dawn (pass the install directory via ``DAWN_ROOT``) and Clang (if CMake has trouble finding Clang or LLVM, set ``LLVM_ROOT`` to the correct directory). Note that Ubuntu 16.04 ships with Clang (3.8.0) and a simple
+Please ensure that dawn already built succesfully before attempting to build gtclang. Then:
 
 ```bash
-sudo apt-get install llvm-3.8-dev clang-3.8
-```
-
-will install it. An example invocation of CMake may look like
-
-```bash
-git clone https://github.com/MeteoSwiss-APN/gtclang.git
-cd gtclang
+cd dawn/gtclang/bundle
 mkdir build && cd build
-cmake .. -DDAWN_ROOT=<dawn-install-dir>
-make -j4
+cmake .. -DProtobuf_DIR=</path/to/protobuf/> -Ddawn_DIR=<path/to/dawn>/dawn/bundle/install/cmake #ADDITIONAL_FLAGS#
```

Comment:
I would make protobuf optional since it would work if the bundle fetches them again

Reply:
I still see the old one?